modules/core/src/test/scala/doobie/util/strategy.scala

```diff
@@ -165,22 +167,6 @@ class strategyspec extends Specification {
 
   }
 
-  "ResultSet safety" >> {
```

Comment:
It doesn't seem safe to me to remove this test - why remove it?

Reply:
Because I removed ReslutSet#close, and it's safe to that, according to jdbc documentation.